DESCRIPTION

```diff
@@ -27,7 +27,7 @@ License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
 Imports: methods, R6, stats, utils, extraDistr (>= 1.9.1)
-Depends: R (>= 3.3.0), AlphaSimR (>= 1.3.2)
+Depends: R (>= 3.3.0), AlphaSimR (>= 1.3.2), RANN, ggplot2
```

Comment:
I think RANN needs to go to imports and lets see how to use base R plotting to have less dependencies